### ISSUE TITLE
How to use Revisiting Oxford and Paris to pre-train model

### ISSUE THREAD

**Issue by pxy522** [cvg/LightGlue#63]
Hi there

Thanks for your great work!

I am a new student in machine learning and am facing difficulties in retraining the model due to issues with data and loss.

1. In section 4, "Details that Matter," the paper discusses using the "Revisiting Oxford and Paris" dataset for pre-training the model. However, since this dataset was designed for retrieval purposes, I am uncertain how to incorporate it into the retraining of the model.

2. How to calculate the matching error of two images after obtaining Superpoints feature points.

**Comment by sarlinpe** [comment]
See https://github.com/cvg/glue-factory. Please open an issue over there if anything remains unclear.
